go.mod

```diff
@@ -75,16 +75,16 @@ require (
 	go.uber.org/zap v1.24.0
 	go4.org/mem v0.0.0-20220726221520-4f986261bf13
 	go4.org/netipx v0.0.0-20230728180743-ad4cb58a6516
-	golang.org/x/crypto v0.18.0
+	golang.org/x/crypto v0.30.0
```

Comment:
I think there is a CVE in 0.30.0 --- can we bump to 0.31.0?

https://go.googlesource.com/crypto/+/refs/tags/v0.31.0

Reply:
Done :+1: